src/resources/views/base/inc/main_header.blade.php

```diff
@@ -1,12 +1,12 @@
 <header class="{{ config('backpack.base.header_class') }}">
   <!-- Logo -->
-  <button class="navbar-toggler sidebar-toggler d-lg-none mr-auto ml-3" type="button" data-toggle="sidebar-show">
+  <button class="navbar-toggler sidebar-toggler d-lg-none mr-auto ml-3" type="button" data-toggle="sidebar-show" aria-label="{{ trans('backpack::base.toggle_navigation')}}">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <a class="navbar-brand" href="{{ url(config('backpack.base.home_link')) }}">
+  <a class="navbar-brand" href="{{ url(config('backpack.base.home_link')) }}" title="{{ config('backpack.base.project_name') }}">
```

Comment:
Looking at it, why the change from `home_link` to `project_name` ? @martijnb92

Reply:
`home_link` is not changed, only a title attribute with the project name is added.

Reply:
Sorry you are right. Read it like :upside_down_face: